Dockerfile

```
@@ -123,11 +126,14 @@ RUN bash -lc "gem update --system && gem install bundler"
 
 # install conan
 RUN apt-get install -y python-dev && \
-	pip install --ignore-installed six --ignore-installed colorama --ignore-installed requests --ignore-installed chardet --ignore-installed urllib3 --upgrade setuptools && \
-    pip install -Iv conan==1.11.2
+	pip install --no-cache --ignore-installed six --ignore-installed colorama \
```

oops @bluefish6, found another one. is this also supposed to be `--no-cache-dir`

---

Of course you're right. Sorry, I should have checked the whole file when I got the first comment.
```suggestion
	pip install --no-cache-dir --ignore-installed six --ignore-installed colorama \
```

This seems to be the last one though ;)